api-reference/listing/delete.mdx

```diff
@@ -77,9 +77,12 @@ Returned if the currently authenticated user does not have permission to delete
 <RequestExample>
 
 ```bash Example Request
-curl --location --request "DELETE 'https://api.travtus.com/listings/listing-id-1?inactive_date=2012-01-23/' \
+curl --location --request "DELETE 'https://api.travtus.com/listings/listing-id-1/' \
 --header 'Content-Type: application/json' \
---header 'Authorization: bearer <token>'
+--header 'Authorization: bearer <token>' \
+--data-raw '{
+  "inactive_date": "2012-01-23"
+}
```

Comment:
Would it make sense to have unit tests for these example requests. Not necessarily an issue for this PR but just wanted your thoughts on it.

Reply:
Yes, for sure, great suggestion!
I have an issue to add unit tests in the Travuts-API repo and I've updated it to mention that we should use these examples in the tests
https://github.com/Travtus/adam-api/issues/12

Reply:
Sounds good, thanks